components/_share.scss

```diff
@@ -108,7 +112,8 @@
   }
 
   .sharing-elsewhere {
-    margin: 20px 0 30px 0;
+    margin: 20px 0 0 0;
```

Comment:
Properties should be ordered border-top, font-size, margin, overflow, padding-top<br>Shorthand form for property `margin` should be written more concisely as `20px 0 0` instead of `20px 0 0 0`